### ISSUE TITLE
Could not start 'command'

### ISSUE THREAD

**Issue by jakob-grabner** [vanniktech/gradle-dependency-graph-generator-plugin#19]
It seems the plugin is unable to detected that I'm using a windows machine and tries to start command instead of cmd. Any ideas why this could happen?

`Caused by: net.rubygrapefruit.platform.NativeException: Could not start 'command'
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:27)
        at net.rubygrapefruit.platform.internal.WindowsProcessLauncher.start(WindowsProcessLauncher.java:22)
        at net.rubygrapefruit.platform.internal.WrapperProcessLauncher.start(WrapperProcessLauncher.java:36)
        at org.gradle.process.internal.ExecHandleRunner.run(ExecHandleRunner.java:68)
`

**Comment by vanniktech** [comment]
Try the 0.3.0-SNAPSHOT which should have fixed this.

```
buildscript {
  repositories {
    maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
  }
  dependencies {
    classpath "com.vanniktech:gradle-dependency-graph-generator-plugin:0.3.0-SNAPSHOT"
  }
}

apply plugin: "com.vanniktech.dependency.graph.generator"
```

**Comment by jakob-grabner** [comment]
Worked with 0.3.0-SNAPSHOT. Thanks!

**Comment by vanniktech** [comment]
I'll release 0.3.0 tonight.

**Comment by vanniktech** [comment]
I just released version 0.3.0 which should be available soon.
